src/_color-themes.scss

```diff
@@ -0,0 +1,120 @@
+    alt-accent-light: $blue-light,
+    alt-accent: $blue-themed,
+    main-accent: $teal,
+    alt-accent-baby: $blue-baby,
+    red-light: $red-light,
+    red-lighter: $red-lighter,
+    gray-shadow: $gray-shadow,
+    white-themed: $white-pure,
+    black-themed: $black-pure,
+    gray-lightest: #f8f9fa,
+    gray-lighter: $gray-200,
+    gray-light: $gray-300,
+    gray-dark: $gray-800,
+    button-border-color: $black-pure,
+    form-control-border: $black-pure,
+    input-group-text-border: #ced4da,
+    select-border: $gray-300,
+    select-box-top: #f8f9fa,
+    select-box-bottom: $gray-200,
+    select-box-top-bevel: $white-pure,
+    table-border: $gray-300,
+    card-bg: $white-pure,
+    card-border: #dfdfdf,
+    card-footer-bg: #f7f7f7,
+    toast-fg: #6c757d,
+    toast-bg: $white-pure,
+    toast-close: $black-pure
+  ),
+  dark: (
+    bg: $gray-darkest,
+    fg: $gray-200,
+    alt-accent-lightest: #121925,
+    alt-accent-lighter: #0f1e38,
+    alt-accent-light: #1761da,
+    alt-accent: #1959c9,
+    main-accent: $teal-dark,
+    alt-accent-baby: $teal,
+    red-light: $red-light,
+    red-lighter: $red-lighter,
+    gray-shadow: $gray-darker,
+    white-themed: #ddd,
+    black-themed: $gray-200,
+    gray-lightest: $gray-darker,
+    gray-lighter: $gray-800,
+    gray-light: $gray-500,
+    gray-dark: $gray-200,
+    button-border-color: $gray-darkest,
+    form-control-border: #888,
+    input-group-text-border: $gray-mid,
+    select-border: $gray-mid,
+    select-box-top: $gray-800,
+    select-box-bottom: $gray-darker,
+    select-box-top-bevel: $gray-darker,
+    table-border: $gray-mid,
+    card-bg: $gray-darkest,
+    card-border: $gray-dark,
+    card-footer-bg: $gray-darker,
+    toast-fg: $gray-200,
+    toast-bg: $gray-800,
+    toast-close: $white-pure
+  )
+);
+
+// credit: https://medium.com/@dmitriy.borodiy/easy-color-theming-with-scss-bc38fd5734d1
+@mixin themed {
+  @each $theme, $map in $themes {
+    .theme--#{$theme} & {
+      $theme-map: () !global;
+
+      @each $key, $submap in $map {
+        $value: map-get(map-get($themes, $theme), "#{$key}");
+        /* stylelint-disable */
+        $theme-map: map-merge(
+          $theme-map,
+          (
+            $key: $value
+          )
+        ) !global;
+      }
+      /* stylelint-enable */
+      @content;
+      $theme-map: null !global;
+    }
+  }
+}
+
+// body rule needs special treatment, although this could probably be heavily simplified
+@mixin themed_body {
+  @each $theme, $map in $themes {
+    .theme--#{$theme} {
+      $theme-map: () !global;
+
+      @each $key, $submap in $map {
+        $value: map-get(map-get($themes, $theme), "#{$key}");
+        /* stylelint-disable */
+        $theme-map: map-merge(
+          $theme-map,
+          (
+            $key: $value
+          )
+        ) !global;
+      }
+      /* stylelint-enable */
+      @content;
+      $theme-map: null !global;
+    }
+  }
+}
+
+@function t($key) {
```

Comment:
Do you mind renaming this to `theme`? I feel like it's going to be pretty confusing to newcomers otherwise.